docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
```

Comment:
<img width="339" alt="ss2" src="https://user-images.githubusercontent.com/60163283/140517626-cbb949d1-7f97-41e2-824f-87ac53d94a2d.PNG">
setIndex(1) <- need the parameter